packages/cli/src/bin/index.ts

```diff
@@ -107,7 +108,28 @@ const cli = async (
     disable
   };
   try {
-    const outcome = await axeTestUrls(urls, testPageConfigParams, events);
+    let outcome;
+    try {
+      outcome = await axeTestUrls(urls, testPageConfigParams, events);
+    } catch (e) {
+      // Provide a more user-friendly error message when there's a ChromeDriver/Chrome version mismatch.
+      if (e instanceof selenium_error.SessionNotCreatedError && e.message.includes(
+        'This version of ChromeDriver only supports'
+        // This string has to match the error message printed by chromedriver, see
+        // https://chromium.googlesource.com/chromium/src/+/refs/tags/110.0.5481.194/chrome/test/chromedriver/chrome_launcher.cc#300.
+      )) {
+        console.error(error('Error: %s'), e.message);
+        console.log(`\nConsider installing a matching version of ChromeDriver with:
+
+    npm install -g chromedriver@<version>
+
+(where <version> is the desired version of Chromedriver.)
+And running \`axe\` with \`--chromedriver-path $(npm root -g)/chromedriver/bin/chromedriver\`.`)
```

Comment:
Just some slight formatting suggestions, trying to make it absolutely clear what is expected of the user and what is needed to be done. Also it would be very helpful to give a full cli example using `chromedriver-path`.

```
Error: session not created: This version of ChromeDriver only supports Chrome version 109
Current browser version is 111.0.5563.64 with binary path /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

Please install a matching version of ChromeDriver and run `axe` using the `--chromedriver-path` argument:

    $ npm install -g chromedriver@<version>
    $ axe --chromedriver-path $(npm root -g)/chromedriver/bin/chromedriver https://google.com https://foo.com

(where <version> is the version of ChromeDriver that matches the current browser's major version)
```

```suggestion
        console.error(error('Error: %s'), e.message);
        console.log(`\nPlease install a matching version of ChromeDriver and run \`axe\` using the \`--chromedriver-path\` argument:

    $ npm install -g chromedriver@<version>
    $ axe --chromedriver-path $(npm root -g)/chromedriver/bin/chromedriver ${urls.join(' ')}

(where <version> is the version of ChromeDriver that matches the current browser's major version)`);
```